docs/building.md

```diff
@@ -0,0 +1,42 @@
+# Building Tofino
+
+## All platforms
+
+Ensure you have the following installed and accessible in your `PATH` environment variable.
+
+* [Node.js](https://nodejs.org) >= 6.1.0
+  * To manage multiple node installations, consider using [nvm](https://github.com/creationix/nvm)
+* [Python 2.7](https://www.python.org/) (`python` must be in path)
+
+### Windows
+
+* Install [Visual Studio 2013](https://www.visualstudio.com/news/vs2013-community-vs)
+* Install [Visual C++ Redistributable for VS2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145), as currently Electron is built with VS2015. After [electron/electron#5528](https://github.com/electron/electron/issues/5528) has been fixed, this is no longer necessary, but we still need to update to Electron 1.1.0 in #517.
```

Comment:
Add a comment in #517 to update this note. Maybe this comment already does that!